server/gfmd/user.h

```diff
@@ -70,3 +70,8 @@ gfarm_error_t gfm_server_user_info_remove(struct peer *, int, int);
 struct group_assignment;
 /* subroutine of grpassign_add(), shouldn't be called from elsewhere */
 void grpassign_add_group(struct group_assignment *);
+
```

Comment:
Please remove the "struct user_auth" line and the empty line,
because we don't have to export struct user_auth.